force-app/main/default/pages/GE_TokenizeCard.page

```diff
@@ -174,17 +175,17 @@
            .then((resp) => {
               if (resp.token) {
                  // Send token to lwc
-                 const successString = JSON.stringify({ type: postMessageType, token: resp.token });
+                 const successString = { type: postMessageType, token: resp.token };
                  sendPostMessage(successString);
 
               } else if (resp.error) {
                  // Response contains the error, send it to lwc
-                 sendPostMessage(JSON.stringify({ type: postMessageType, ...resp }));
+                 sendPostMessage(({ type: postMessageType, ...resp }));
               }
            })
            .catch((err) => {
               // Send error to lwc
-              const errorString = JSON.stringify({ type: postMessageType, ...err });
+              const errorString = ({ type: postMessageType, ...err });
               sendPostMessage(errorString);
```

Comment:
Minor thing, but `sendPostMessage` below this should have its param description updated.